dbms/src/Storages/DeltaMerge/Remote/DataStore/DataStore.h

```diff
@@ -32,7 +32,7 @@ class IPreparedDMFileToken : boost::noncopyable
     /**
      * Restores into a DMFile object. This token will be kept valid when DMFile is valid.
      */
-    virtual DMFilePtr restore(DMFileMeta::ReadMode read_mode) = 0;
+    virtual DMFilePtr restore(DMFileMeta::ReadMode read_mode, UInt32 meta_version) = 0;
```

Comment:
Is meta_version defined as UInt64 rather than UInt32?

Reply:
Fixed